pre-docs/inventory_version2.md

```diff
@@ -82,17 +94,89 @@ groups:
       transport: ssh
 ```
 
-`target-lookups` contains an array of target lookup objects. Each
-`target-lookup` entry must include a `plugin` key that defines which plugin
-should be used for the lookup. The rest of the keys are specific to the plugin
-being used.
+Make sure you have [configured PuppetDB](./bolt_connect_puppetdb.md)
+
+#### Terraform
 
-For the puppetdb plugin The query field is a string containing a pql query or an array containing a
-query in the puppetdb ast syntax.
+The Terraform plugin accepts several fields:
 
-Make sure you have [configured puppetdb](./bolt_connect_puppetdb.md)
+`dir`: The directory from which to load Terraform state
```

Comment:
Ah - sorry I think this also needs to be a list. The rich diff shows the options in just a paragraph